cmd/memcached_exporter/main.go

```diff
@@ -36,6 +39,12 @@ func main() {
 		address       = kingpin.Flag("memcached.address", "Memcached server address.").Default("localhost:11211").String()
 		timeout       = kingpin.Flag("memcached.timeout", "memcached connect timeout.").Default("1s").Duration()
 		pidFile       = kingpin.Flag("memcached.pid-file", "Optional path to a file containing the memcached PID for additional metrics.").Default("").String()
+		enableTls     = kingpin.Flag("tls.enable", "Enable TLS connections to memcached").Bool()
```

Comment:
Rather than have a bool, we can check for `keyfile != ""`. If there's a key, enable TLS.

Reply:
I did consider just checking for a cert flag, but those are for MTLS, while just enabling with `--tls.enable` and an optional `--tls.cafile` let you do plain TLS.